api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/bash
 
-# This generates big query count databases cdr and public that get put in cloudsql for workbench and data browser
+# This generates big query count databases cdr that get put in cloudsql for workbench
 
 set -xeuo pipefail
 IFS=$'\n\t'
 
-
 # get options
 # --project=all-of-us-workbench-test *required
 
 # --cdr=cdr_version ... *optional
-USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT>"
+USAGE="./generate-clousql-cdr/make-bq-data.sh --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT> --public-project <PROJECT>"
```

Comment:
Half an existing issue, but why aren't --workbench-dataset and --public-dataset are missing from the usage docs

Reply:
Changed it.